### PR TITLE
migration to VAS

### PR DESCRIPTION
**Pull request by aameijide**

### CODE REVIEW

File: LongoMatch.Core/Common/EventsBrokerLongomatch.cs [hunk 1]
```diff
@@ -40,15 +42,13 @@ public class EventsBroker
 		public event EventEditedHandler EventEditedEvent;
 		public event MoveEventHandler MoveToEventTypeEvent;
 		public event TimeNodeChangedHandler TimeNodeChanged;
-		public event TimeNodeStartedHandler TimeNodeStartedEvent;
-		public event TimeNodeStoppedHandler TimeNodeStoppedEvent;
+		//		public event TimeNodeStartedHandler TimeNodeStartedEvent;
+		//		public event TimeNodeStoppedHandler TimeNodeStoppedEvent;
```
Comment (vmartos) [inline LongoMatch.Core/Common/EventsBrokerLongomatch.cs]: Code commented, remove if it's not necessary anymore, or explain why it's commented.

File: LongoMatch.Core/Common/EventsBrokerLongomatch.cs [hunk 2]
```diff
@@ -468,25 +474,25 @@ public void EmitMigrateDB ()
...
+		//			}
+		//		}
+		//
+		//		public void EmitTimeNodeStoppedEvent (TimeNode node, TimerButton btn, List<DashboardButton> from)
+		//		{
+		//			if (TimeNodeStoppedEvent != null) {
+		//				if (from == null)
+		//					from = new List<DashboardButton> ();
+		//				TimeNodeStoppedEvent (node, btn, from);
+		//			}
```
Comment (vmartos) [inline LongoMatch.Core/Common/EventsBrokerLongomatch.cs]: The same, code commented, remove it or explain why it's commented

File: LongoMatch.Core/ConfigLongoMatch.cs [hunk 3]
```diff
@@ -256,17 +191,17 @@ public static void Save ()
...
-		public static string DBDir {
-			get {
-				return Path.Combine (homeDirectory, "db");
-			}
-		}
+		//		public static string DBDir {
+		//			get {
+		//				return Path.Combine (homeDirectory, "db");
+		//			}
+		//		}
```
Comment (vmartos) [inline LongoMatch.Core/ConfigLongoMatch.cs]: Remove comments

File: LongoMatch.Core/Interfaces/ITemplatesLongoMatch.cs [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ﻿//
-//  Copyright (C) 2015 Fluendo S.A.
+//  Copyright (C) 2016 dfernandez
```
Comment (vmartos) [inline LongoMatch.Core/Interfaces/ITemplatesLongoMatch.cs]: Change to Copyrigth (C) Fluendo S.A.

File: LongoMatch.Core/LongoMatch.Core.Net45.csproj [hunk 5]
```diff
@@ -40,23 +40,32 @@
...
-    <Reference Include="PropertyChanged">
-      <HintPath>$(SolutionDir)\packages\PropertyChanged.Fody.1.50.3\Lib\portable-net4+sl4+wp8+win8+wpa81+MonoAndroid16+MonoTouch40\PropertyChanged.dll</HintPath>
-    </Reference>
     <Reference Include="Newtonsoft.Json">
       <HintPath>$(SolutionDir)\packages\Newtonsoft.Json.7.0.1\lib\net45\Newtonsoft.Json.dll</HintPath>
       <Package>newtonsoft-json</Package>
     </Reference>
+    <Reference Include="PropertyChanged">
+      <HintPath>..\oneplay-longomatch\packages\PropertyChanged.Fody.1.50.3\Lib\portable-net4+sl4+wp8+win8+wpa81+MonoAndroid16+MonoTouch40\PropertyChanged.dll</HintPath>
+      <Private>False</Private>
```
Comment (vmartos) [inline LongoMatch.Core/LongoMatch.Core.Net45.csproj]: This makes reference to oneplay-longomatch path, if you open the solution Longomatch (openSource) without cloning oneplay-longomatch breaks the PropertyChanged and he can't find the assembly, use $(SolutionDir)

File: LongoMatch.Core/ConfigLongoMatch.cs [hunk 6]
```diff
@@ -159,19 +157,7 @@ public static void Save ()
...
-			}
-			set {
-				style = value;
-			}
-		}
-
-		public static string ConfigFile {
+		new public static string ConfigFile {
 			get {
 				string filename = Constants.SOFTWARE_NAME.ToLower () + "-1.0.config";
```
Comment (turran) [inline LongoMatch.Core/ConfigLongoMatch.cs]: What is the reason for the -1.0? is it the software version? if so, can it be fetched from the software major version number?

Comment (ylatuya) [inline LongoMatch.Core/ConfigLongoMatch.cs]: It's there to make the difference between longomatch 0.x and longomatch 1.x, the versión number should remain the same.

File: LongoMatch.Core/Interfaces/Multimedia/IFramesCapturer.cs [hunk 7]
```diff
@@ -17,20 +17,17 @@
...
 namespace LongoMatch.Core.Interfaces.Multimedia
 {
-
-
 	public interface IFramesCapturer
 	{
 		bool Open (string uri);
-		void Dispose();
-		Image GetFrame (Time pos, bool accurate, int outwidth=-1, int outheight=-1);
+
```
Comment (turran) [inline LongoMatch.Core/Interfaces/Multimedia/IFramesCapturer.cs]: Create the missing documentation on interfaces

File: LongoMatch.Core/Interfaces/Multimedia/IRemuxer.cs [hunk 8]
```diff
@@ -24,10 +23,9 @@ public interface IRemuxer
 	{
 		event ErrorHandler Error;
```
Comment (turran) [inline LongoMatch.Core/Interfaces/Multimedia/IRemuxer.cs]: Missing documentation

File: LongoMatch.Core/Catalog.cs [hunk 9]
```diff
@@ -1,56 +0,0 @@
...
-		public static String GetString (String s)
-		{
-			return Mono.Unix.Catalog.GetString(s);
-		}
-
-		public static String GetPluralString (String s, String p, Int32 n)
-		{
-			return Mono.Unix.Catalog.GetPluralString(s, p, n);
-		}
-	}
```
Comment (vguzmanp) [inline LongoMatch.Core/Catalog.cs]: The Longomatch Catalog must be kept here. VAS will have a different one ([ESA-72](https://github.com/fluendo/VAS/tree/ESA-72))

File: LongoMatch.Core/ConfigLongoMatch.cs [hunk 10]
```diff
@@ -418,28 +304,7 @@ public static string RelativeToPrefix (string relativePath)
 			set {
 				state.captureEncodingProfile = value;
 				Save ();
-				
-			}
-		}
 
-		public static VideoStandard RenderVideoStandard {
```
Comment (vguzmanp) [inline LongoMatch.Core/ConfigLongoMatch.cs]: Why is `RenderVideoStandard` moved to VAS but `CaptureVideoStandard`, etc aren't?

File: LongoMatch.Core/ConfigLongoMatch.cs [hunk 11]
```diff
@@ -609,59 +424,14 @@ public static string RelativeToPrefix (string relativePath)
 	}
 
 	[Serializable]
-	//[JsonConverter (typeof (LongoMatchConverter))]
-	public class ConfigState
+	[JsonConverter (typeof(LongoMatchConverter))]
+	public class ConfigState : VAS.ConfigState
```
Comment (vguzmanp) [inline LongoMatch.Core/ConfigLongoMatch.cs]: Do we want it to have the same name?

File: LongoMatch.Core/FodyWeavers.xml [hunk 12]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Weavers>
-  <PropertyChanged  InjectOnPropertyNameChanged='false' />
-  <Loader />
+	<PropertyChanged InjectOnPropertyNameChanged="false" />
+	<Loader />
```
Comment (vguzmanp) [inline LongoMatch.Core/FodyWeavers.xml]: @aameijide You probably have Xamarin Studio configured differently, and it's modifying the indentation

Comment (vguzmanp) [inline LongoMatch.Core/FodyWeavers.xml]: Also other FodyWeavers have `'false', not `"false"`

File: LongoMatch.Core/Handlers/MultimediaLongoMatch.cs [hunk 13]
```diff
@@ -34,7 +33,7 @@ namespace LongoMatch.Core.Handlers
 	public delegate void PrevButtonClickedHandler ();
 	public delegate void ProgressHandler (float progress);
 	public delegate void FramesProgressHandler (int actual,int total,Image frame);
-	public delegate void DrawFrameHandler (TimelineEvent play,int drawingIndex,CameraConfig camConfig,bool current);
+	public delegate void DrawFrameHandler (TimelineEventLongoMatch play,int drawingIndex,CameraConfig camConfig,bool current);
 	public delegate void ElapsedTimeHandler (Time ellapsedTime);
 	public delegate void PlaybackRateChangedHandler (float rate);
 	public delegate void SeekHandler (SeekType type,Time start,float rate);
```
Comment (vguzmanp) [inline LongoMatch.Core/Handlers/MultimediaLongoMatch.cs]: SeekHandler is defined in both VAS and LongoMatch

**Comment by vguzmanp** [comment]
Pending Renames:
Classes & files:
    •    ActionLinkLongoMatch -> TeamActionLink
    •    ProjectLongoMatch -> TeamSportProject
    •    TimelineEventLongoMatch -> TeamSportTimelineEvent
    •    TimerLongoMatch -> TeamTimer

Files:
    •    EventTypeLongoMatch.cs -> SportEventTypes.cs
    •    DashboardButtonLongoMatch.cs -> SportDashboardButtons.cs

In doubt:
    •    ConfigLongoMatch -> Config?
    •    PlayerLongoMatch -> Person? TeamPlayer? Other?

**Pull request merged**